docs/sdks/stash/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
```

Comment:
Where is this page linked from? Should we have a mobile app integration guide page on these mobile sdk/GMA page? Or should we start small and just add on
https://unifiedid.com/docs/overviews/overview-publishers

Under resources section to say for mobile app integration, check the mobile sdk/gma guide?

Reply:
@sunnywu: Noted for GMA and IMA docs which are not in this PR. Note: I didn't put the full name in the title -- it's too long. But it's in the first line / first occurrence of the term in the doc, for both IMA and GMA.